app/forms/idv/profile_form.rb

```diff
@@ -35,11 +38,11 @@ def profile
     end
 
     def pii_attributes
-      @_pii_attributes ||= Pii::Attributes.new
+      @_pii_attributes ||= Pii::Attributes.new_from_hash(previous_address: OpenStruct.new)
```

Comment:
I was optimizing for developer time and grokability, rather than performance.

I find these 2 lines different in their readability:
```ruby
previous_address[new_key] = value                               # easier
previous_address.public_send("#{new_key}=", value)  # harder
```